### ISSUE TITLE
Problem about load label

### ISSUE THREAD

**Issue by shanqiu24** [2gunsu/monocon-pytorch#11]
When I load the original label, It will read each data from the __getitem__ function during each epoch training right?but I don't understand the "raw_labels.convert_cam (src_cam=0, dst_cam=2)" in this method, which seems to be calibrated according to the P0 projection matrix and the P2 projection matrix? But my dataset does not provide P0 projection matrix, so can I omit this step to not processing the raw_labels ?😂

**Comment by 2gunsu** [comment]
Since the raw annotation (.txt file) of the KITTI dataset is based on camera 0 and the image was taken by camera 2, it changes the coordinates from camera 0 to camera 2.

Therefore, if the dataset you want to use is already aligned with the image and ground truth, you do not need a code such as ```raw_labels.convert_cam (src_cam=0, dst_cam=2)```.
